pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -220,22 +227,17 @@ class (CLASSNAME argument) in the specified namespace. If no namespace was
 @class_group.command('references', options_metavar=CMD_OPTS_TXT)
 @click.argument('classname', type=str, metavar='CLASSNAME', required=True)
 @click.option('-R', '--result-class', type=str, required=False,
-              metavar='<class name>',
-              help='Filter by the classname provided. Each returned '
-                   'class (or classname) should be this class or its '
-                   'subclasses. Optional.')
+              metavar='CLASSNAME',
+              help='Filter the result set by result class name.')
```

Comment:
I know, but I find the original text to be too detailed so that the point gets lost in the details ("result class"), and your new suggestion is somewhat ambiguous to my reading as to where the subclassing is applied (to the class in the result set,m or to the specified class).

I think we should separate the statements about the main point ("result class") and how the detail about subclasses works.
How about: "Filter the result set by result class name. Subclasses of the specified class also match."

Reply:
DONE - Added "Subclasses of the specified class also match." in all cases of class name based filtering for classes and instances.